examples/zcEarly_scenario.hs

```diff
@@ -25,6 +27,6 @@ circuit uv oc zc gp_ack gn_ack gp gn =
     gnHandshake = handshake gn gn_ack
 
     initialState = initialise uv False <> initialise oc False
-
-    chargeFunc = ocFunc <> ocReact <> environmentConstraint <> circuitConstraint 
-                 <> gpHandshake <> gnHandshake <> initialState
+    
```

Comment:
I noticed them and fixed them right after commit. Oh dear. I use sublime text, do you know if this is possible? And if so, how?